src/app/commons/Button.tsx

```diff
@@ -0,0 +1,49 @@
+import { ButtonProps } from "../types/types";
+
+const Button: React.FC<ButtonProps> = ({
+    type,
+    fullWidth,
+    children,
+    onClick,
+    secondary,
+    danger,
+    disabled,
+  }) => {
+    let buttonClasses = `flex justify-center rounded-md px-3 py-2 text-sm font-semibold focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2`;
```

Comment:
Why are we using backticks here? We're not using template string.

Reply:
Issue addressed in commit 81c2ee1cc5605f1af059f2304ef79d6902f9374d, which refactors the button class name.